x-pack/plugins/alerts/server/alerts_client/lib/map_sort_field.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+const sortFieldMap: Record<string, string> = {
+  name: 'name.keyword',
+};
+
+export function mapSortField(field?: string): string | undefined {
+  return field ? sortFieldMap[field] || field : undefined;
+}
```

Comment:
To de-couple our mappings from HTTP API parameters, I decided to create this function, so people don't have to know they need to pass `name.keyword` when they want to sort by `name`.